src/components/MessagePanel/index.jsx

```diff
@@ -17,53 +17,65 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-export default function ErrorPanel({
```

Comment:
Instead of re-writing & renaming the ErrorPanel, let's just create a separate MessagePanel. I created https://github.com/mozilla-frontend-infra/balrog-ui/pull/232 which we could use. @allan-silva Thoughts?

Reply:
That is great, thanks!